packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -160,6 +160,7 @@ const schema: SchemaType<"Tags"> = {
       foreignTypeName: "TagRel",
       foreignFieldName: "tagId",
       //filterFn: tagRel => tagRel.baseScore > 0, //TODO: Didn't work with filter; votes are bypassing the relevant callback?
+      filterFn: tagRel => !tagRel.deleted // TODO: per the above, we still need to make this check baseScore > 0
```

Comment:
This should have been the case already, and is needed to be consistent with the behaviour in `mergeTags` (where we decrement postCount by a specific amount)